docs/articles_en/assets/snippets/ov_stateful_model_intro.py

```diff
@@ -0,0 +1,208 @@
+# Copyright (C) 2018-2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging as log
+import numpy as np
+
+import openvino  as ov
+from openvino.runtime import opset13 as ops
+from openvino.runtime.op.util import VariableInfo, Variable
+from openvino.runtime.passes import LowLatency2, MakeStateful, Manager
+from openvino.runtime.utils import replace_node
+
+
+def state_network_example():
+    #! [ov:state_network]
+    input = ops.parameter([1, 1], dtype=np.float32, name="data")
+    init_const = ops.constant([[0]], dtype=np.float32)
+
+    # The ReadValue/Assign operations must be used in pairs in the network.
```

Comment:
```suggestion
    # The ReadValue/Assign operations must be used in pairs in the model.
```

Reply:
"# The ReadValue/Assign operations must be used in pairs in the network." limitation is no longer relevant
Maybe we can write something like that 
"Typically ReadValue/Assign operations are presented as pairs in models. ReadValue reads information from an internal memory buffer, Assign operation writes data to this buffer. For each  pair, its own Variable object must be created. Variable defines name, shape and type of the buffer."

Reply:
updated for c++ snippet as well